rest_api/packaging/ubuntu/postinst

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+user="sawtooth"
+group="sawtooth"
+
+if ! getent passwd sawtooth > /dev/null; then
```

Comment:
Should the getent tests in all the postinst scripts use the vars set above instead of "sawtooth"?